app/src/main/res/layout/activity_add_prompt.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".AddPrompt">
+
+    <TextView
```

Comment:
We need a scroll view here because if our prompt is too long, we can't see the buttons

Reply:
Look into recycle view for android for making lazy lists

Reply:
Ideally we should have one fragment for "all prompts", "add prompt", "edit prompt", "prompt detail" because we might have long ones and want to see more details

All prompts should be a list in recycle view, showing the first few lines

We then select/delete/edit the prompt by swipe or something, or we can do it in the details fragment